<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>par3_2</class>
 <widget class="QDialog" name="par3_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerer les reservations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>noPlace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="L_noPlace"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>noRang</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="L_noRang"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>dateRep</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="L_dateRep"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>noDos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="L_noDos"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>catR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="L_catR"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="text">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_2">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>par3_2</receiver>
   <slot>ajouter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delete_2</sender>
   <signal>clicked()</signal>
   <receiver>par3_2</receiver>
   <slot>delete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
  <slot>ajouter()</slot>
  <slot>delete()</slot>
 </slots>
</ui>
